File: ms-auth-java/domain/model/src/main/java/co/com/bancolombia/model/session/Session.java
package co.com.bancolombia.model.session;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@Builder(toBuilder = true)
public class Session {
    private String sessionId;
    private String email;
}

File: ms-auth-elixir/lib/domain/model/bancolombia/session/session.ex
defmodule Authelixir.Domain.Model.Bancolombia.Session.Session do
  @moduledoc """
  Sesión de autenticación.
  """

  defstruct [:session_id, :email]

  @type t :: %__MODULE__{
          session_id: String.t(),
          email: String.t()
        }
end
